src/components/create-job/index.js

```diff
@@ -11,11 +11,11 @@ import * as api from '../../api';
 import '../common.scss';
 import './index.scss';
 
-const defaultMap = `function map(key, value) {
+const defaultMap = `function map(data, emit) {
```

Comment:
Any reason this can't be key/val? That is more self-explanatory and matches the general MapReduce setup

Reply:
There isn’t really a key value pair coming into a map function. It’s just data it could be text csv line or other stuff